src/status_im2/contexts/wallet/home/style.cljs

```diff
@@ -23,3 +25,13 @@
   {:justify-content :center
    :flex            1
    :margin-bottom   44})
+
+(def separator
+  {:width 12})
+
+(def selected-tab-container
+  {:padding-horizontal 8})
+
+(def home-container
+  {:margin-top (safe-area/get-top)
```

Comment:
Can this safe area margin be applied in the navigation file?

Reply:
Probably yes, but I'd better make it the scope of other PR